tests/components/hunterdouglas_powerview/test_config_flow.py

```diff
@@ -0,0 +1,220 @@
+"""Test the Logitech Harmony Hub config flow."""
+import asyncio
+import json
+
+from asynctest import CoroutineMock, MagicMock, patch
+
+from homeassistant import config_entries, setup
+from homeassistant.components.hunterdouglas_powerview.const import DOMAIN
+
+from tests.common import load_fixture
+
+
+def _get_mock_powerview_userdata(userdata=None, get_resources=None):
+    mock_powerview_userdata = MagicMock()
+    if not userdata:
+        userdata = json.loads(load_fixture("hunterdouglas_powerview/userdata.json"))
+    if get_resources:
+        type(mock_powerview_userdata).get_resources = CoroutineMock(
+            side_effect=get_resources
+        )
+    else:
+        type(mock_powerview_userdata).get_resources = CoroutineMock(
+            return_value=userdata
+        )
+    return mock_powerview_userdata
+
+
+async def test_user_form(hass):
+    """Test we get the user form."""
+    await setup.async_setup_component(hass, "persistent_notification", {})
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN, context={"source": config_entries.SOURCE_USER}
+    )
+    assert result["type"] == "form"
+    assert result["errors"] == {}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5444210)